src/rich-text-editor.component.js

```diff
@@ -75,7 +75,7 @@ export const RichTextEditor = forwardRef((props, editorRef) => {
   useEffect(() => {
     // Clicking on bandicoot richtext buttons triggers a blur event that will setFocus to false we want to delay the
     // save event that is triggered by blur events. 100ms is arbitrary. Whenever react rerenders the rich-text-editor
-    // due to focused state changing we need to either clear the blurTimeout to prevent a save action from firing 
```

Comment:
my bad, had to change my editor because I was tired of other people on my team constantly fixing my extra spaces 🤷‍♂